package.json

```diff
@@ -83,7 +83,7 @@
     "webpack": "1.13.1"
   },
   "dependencies": {
-    "@okta/okta-auth-js": "1.8.0",
+    "@okta/okta-auth-js": "1.9.0-beta.6",
```

Comment:
Will be changed once https://oktainc.atlassian.net/browse/OKTA-147445 is resolved